src/main/java/com/fibermc/essentialcommands/config/EssentialCommandsConfigSnapshot.java

```diff
@@ -75,6 +75,7 @@ public final class EssentialCommandsConfigSnapshot {
     public final Expression<RespawnCondition> RESPAWN_AT_EC_SPAWN;
     public final boolean PERSIST_BACK_LOCATION;
     public final boolean RECHECK_PLAYER_ABILITY_PERMISSIONS_ON_DIMENSION_CHANGE;
+    public final int FLY_MAX_SPEED;
```

Comment:
Curiosity: any reason to not let this be a decimal? (is the vanilla value an int?)

Reply:
Vanilla is decimal, default is 0.05, in code it's now considered in integers of the ingame values multiplied by 20

Reply:
okok, works for me